TLM/TLM/Util/PriorityRoad.cs

```diff
@@ -51,11 +119,51 @@ public static class PriorityRoad {
             ushort segmentId = data.CurSeg.segmentId;
             foreach (bool startNode in Constants.ALL_BOOL) {
                 ushort nodeId = netService.GetSegmentNodeId(segmentId, startNode);
-                FixJunction(nodeId);
+                FixHighPrioirtyJunction(nodeId);
             }
             return true;
         }
 
+        /// <summary>
+        /// returns the node of <paramref name="segmentId"/> that is not shared 
+        /// with <paramref name="otherSegmentId"/> .
+        /// </summary>
+        /// <param name="segmentId"></param>
+        /// <param name="otherSegmentId"></param>
+        /// <returns></returns>
+        private static ushort GetUnsharedNode(ushort segmentId, ushort otherSegmentId) {
```

Comment:
Naming question. `Unshared` means that it was shared and now is not shared anymore. I.e. something has been done on that shared node. Maybe `GetSharedOrTheOtherNode`?